src/node.rs

```diff
@@ -1223,6 +1223,25 @@ impl<S: ForkSource + std::fmt::Debug> InMemoryNode<S> {
         }
     }
 
+    fn validate_tx(&self, tx: &L2Tx) -> Result<(), String> {
+        let max_gas = U256::from(u32::MAX);
+        if tx.common_data.fee.gas_limit > max_gas
+            || tx.common_data.fee.gas_per_pubdata_limit > max_gas
+        {
+            return Err("exceeds block gas limit".into());
+        }
+
+        if tx.common_data.fee.max_fee_per_gas < tx.common_data.fee.max_priority_fee_per_gas {
```

Comment:
We should also include the check for `max_fee_per_gas` not being too low: 

```rust
if tx.common_data.fee.max_fee_per_gas < L2_GAS_PRICE.into() {
    tracing::info!(
        "Submitted Tx is Unexecutable {:?} because of MaxFeePerGasTooLow {}",
        tx.hash(),
        tx.common_data.fee.max_fee_per_gas
    );
    return Err("max fee per gas less than block base fee".into());
}
```

Reply:
added